theano/gpuarray/opt.py

```diff
@@ -97,6 +95,15 @@ def f(local_opt):
     return f
 
 
+def register_opt2(tracks, *tags, **kwargs):
+    def f(local_opt):
+        name = (kwargs and kwargs.pop('name')) or local_opt.__name__
+        opt = theano.gof.local_optimizer(tracks)(local_opt)
+        gpu_optimizer2.register(name, opt, 'fast_run', 'gpuarray', *tags)
```

Comment:
Why are we registering this as only `fast_run` ? I think it should also be `fast_run` and `fast_compile` as we'd be profiling with `fast_compile`